packages/lib/src/prod/shared-production.ts

```diff
@@ -70,13 +70,23 @@ export function prodSharedPlugin(
           }
         }
         if(module){
+          ${
+            builderInfo.isSystemjs
+              ? ''
+              : 'module = module.default ? module.default : module'
+          }
           moduleCache[name] = module;
           return module;
         }
       }
       async function getSharedFromLocal(name , shareScope) {
         if (moduleMap[name]?.import) {
           const module = (await moduleMap[name].get())()
```

Comment:
Should this be `let module = ...`, so the new change can actually reassign to `module = ...`?

Reply:
Thank you for the review. But in reality it is not necessary. There is still some work to be done on the systemjs case to pass the test.